bin/torii/src/main.rs

```diff
@@ -109,6 +109,10 @@ struct Args {
     /// Chunk size of the events page when indexing using events
     #[arg(long, default_value = "1000")]
     events_chunk_size: u64,
+
+    /// Should cors be disabled
+    #[arg(long)]
+    no_cors: bool,
```

Comment:
I think we should rather make `allowed_origins: Vec<String>` `allowed_origins: Option<Vec<String>>` and CORS should be disabled by default